lib/cartopy/io/img_tiles.py

```diff
@@ -350,23 +355,33 @@ class Stamen(GoogleWTS):
     """
 
     def __init__(self, style='toner',
-                 desired_tile_form=None, cache=False):
-
+                 desired_tile_form=None, cache=False,
+                 apikey=None):
+        if apikey is None:
+            raise ValueError("An apikey is now required to use Stamen maps. "
+                             "Visit https://maps.stadiamaps.com/ for more details.")
```

Comment:
Dead URL? Suggest instead: https://docs.stadiamaps.com/authentication/

Reply:
Yes, I can confirm this is the correct URL. `maps.stadiamaps.com` is not a valid URL.